lib/multipartform.js

```diff
@@ -0,0 +1,16 @@
+var fs = require('fs');
+
+function File(path, filename, contentType) {
+  this.buffer = fs.readFileSync(path);;
+  this.filename = filename || this._basename(path);
+  this.content_type = contentType || "application/octet-stream";
+}
+
+File.prototype._basename = function (path) {
+  var parts = path.split(/\/|\\/);
+  return parts[parts.length - 1];
+};
+
+exports.file = function (path, filename, fileSize, encoding, contentType) {
+  return new File(path, filename, fileSize, encoding, contentType);
```

Comment:
Does `new File` correspond to the constructor function above? It's missing parameters that you're passing here...do we need them?

Also `File` is a web standard, I don't know if node is ever going to support it but maybe a different name might save us for any type of future clashes...maybe `Mulitpart`/`MultipartFile` since that's the name of the file.

Reply:
Good catch, I should get that fixed

Yeah, i can switch the name

Reply:
Updated